src/plugins/discover/public/application/angular/context/api/anchor.test.ts

```diff
@@ -114,6 +116,28 @@ describe('context app', function () {
       });
     });
 
+    it('should update search source correctly when useFieldApi set to false', function () {
+      const searchSource = updateSearchSource(
+        savedSearchMock.searchSource,
+        'id',
+        [],
+        false,
+        indexPatternMock
+      );
+      expect(searchSource.getSearchRequestBody()).toMatchSnapshot();
+    });
+
+    it('should update search source correctly when useFieldApi set to true', function () {
```

Comment:
nit: `useNewFieldsApi` 😬

Reply:
true, thought it's not that new, but didn't dare to name it useOldFieldsApi 👍